.github/workflows/ci.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       - uses: olafurpg/setup-scala@v7
         with:
           java-version: ${{ matrix.java }}
-      - run: git fetch --tags
```

Comment:
does it always complain, or only when the tag points to a different sha1?

Reply:
I couldn't find the exact cause. It works well on fork's CI, but fails in the main repo. Perhaps it related to merged commits from forks